istioctl/cmd/convert_ingress.go

```diff
@@ -97,7 +97,8 @@ func readConfigs(readers []io.Reader) ([]model.Config, []*v1beta1.Ingress, error
 		recognized := 0
 		for _, nonIstio := range kinds {
 			if nonIstio.Kind == "Ingress" &&
-				nonIstio.APIVersion == "extensions/v1beta1" {
+				(nonIstio.APIVersion == "extensions/v1beta1" ||
```

Comment:
Still keep support for extensions in cli?

Reply:
it's basically free so I don't see why we shouldn't, any reasons I am missing?

Reply:
Can come up with one point: to keep consistent